oletools/util/log_helper/_logger_adapter.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from . import _root_logger_wrapper
+
+
+class OletoolsLoggerAdapter(logging.LoggerAdapter):
+    """
+    Adapter class for all loggers returned by the logging module.
+    """
+    _json_enabled = None
+
+    def print_str(self, message):
+        """
+        This function replaces normal print() calls so we can format them as JSON
+        when needed or just print them right away otherwise.
+        """
+        if self._json_enabled and self._json_enabled():
+            # Messages from this function should always be printed,
+            # so when using JSON we log using the same level that set
+            self.log(_root_logger_wrapper.level(), message)
```

Comment:
When running scripts, the banner is always printed. Since when using JSON we are logging, we need to make sure the banner will always be logged, so we use the same level that is set in the root logger.